Source/Core/CesiumTerrainProvider.js

```diff
@@ -76,7 +75,9 @@ define([
      *     url : '//assets.agi.com/stk-terrain/world',
      *     requestVertexNormals : true
      * });
-     *
+     * 
+     * @see TerrainProvider
```

Comment:
`@see` should go at the very bottom of the reference doc here.  Currently, it is in the middle of the example code.